WORKSPACE

```diff
@@ -210,6 +216,12 @@ go_repository(
     # code.
     remote = "https://github.com/prysmaticlabs/bazel-go-ethereum",
     vcs = "git",
+    patch_args = ["-p1"],
+    patches = [
+        # TODO remove patch once https://github.com/prysmaticlabs/bazel-go-ethereum/pull/13 is merged
```

Comment:
Let's block this PR until https://github.com/prysmaticlabs/bazel-go-ethereum/pull/13 is merged, if that's OK.

I'll review that tomorrow.